init-container/Dockerfile

```diff
@@ -0,0 +1,170 @@
+######### python Native Community Agent #########
+
+FROM python:3.11.9 AS python-builder
+ARG ODIGOS_VERSION
+WORKDIR /python-instrumentation
+COPY agents/python ./agents/configurator
+RUN pip install ./agents/configurator/  --target workspace
+RUN echo "VERSION = \"$ODIGOS_VERSION\";" > /python-instrumentation/workspace/initializer/version.py
+
+######### Node.js Native Community Agent #########
+#
+# The Node.js agent is built in multiple stages so it can be built with either upstream
+# @odigos/opentelemetry-node or with a local clone to test changes during development.
+# The implementation is based on the following blog post:
+# https://www.docker.com/blog/dockerfiles-now-support-multiple-build-contexts/
+
+# The first build stage 'nodejs-agent-clone' clones the agent sources from github main branch.
+FROM alpine AS nodejs-agent-clone
+RUN apk add git
+WORKDIR /src
+ARG NODEJS_AGENT_VERSION=main
+RUN git clone https://github.com/odigos-io/opentelemetry-node.git && cd opentelemetry-node && git checkout $NODEJS_AGENT_VERSION
+
+# The second build stage 'nodejs-agent-src' prepares the actual code we are going to compile and embed in odiglet.
+# By default, it uses the previous 'nodejs-agent-clone' stage, but one can override it by setting the
+# --build-context nodejs-agent-src=../opentelemetry-node flag in the docker build command.
+# This allows us to use the agent sources and test changes during development.
+# The output of this stage is the resolved source code to be used in the next stage.
+FROM scratch AS nodejs-agent-src
```

Comment:
this was built for the odiglet. we can reuse it here also but it requires changes to the makefile :)